test/uk/gov/hmrc/hec/services/TaxCheckServiceImplSpec.scala

```diff
@@ -82,9 +82,9 @@ class TaxCheckServiceImplSpec extends AnyWordSpec with Matchers with MockFactory
   def mockTimeProviderToday(d: LocalDate) = (mockTimeProvider.currentDate _).expects().returning(d)
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
-
-  private val now   = TimeUtils.now()
-  private val today = TimeUtils.today()
+  val zonedDateTimeNow           = ZonedDateTime.of(2021, 10, 9, 9, 12, 34, 0, ZoneId.of("Europe/London"))
```

Comment:
if this is used only for the start date, maybe we should call it `startDate` or something like that?